openlibrary/plugins/openlibrary/js/loans/index.js

```diff
@@ -0,0 +1,11 @@
+export function initLoans() {
+  window.q.push(function() {
```

Comment:
As mentioned [here](https://github.com/internetarchive/openlibrary/issues/4474) please `Replace any references to window.q.push with $(fn)`

Reply:
Thank you for the reminder! I'll make that change right now.